jobs/kola-openstack.Jenkinsfile

```
@@ -43,97 +41,105 @@ properties([
+                openstack_image_filepath = openstack_image_filepath[0..-4]
+                // Use a consistent image name for this stream in case it gets left behind
+                openstack_image_name = "kola-fedora-coreos-${params.STREAM}-${params.ARCH}"
 
-        stage('Upload/Create Image') {
-            // Create the image in OpenStack
-            shwrap("""
-            # First delete it if it currently exists, then create it.
-            ore openstack --config-file=\${OPENSTACK_KOLA_TESTS_CONFIG}/config \
-                 delete-image --id=${openstack_image_name} || true
-            ore openstack --config-file=\${OPENSTACK_KOLA_TESTS_CONFIG}/config \
-                 create-image --file=${openstack_image_filepath} \
-                 --name=${openstack_image_name} --arch=${params.ARCH}
-            """)
-        }
-        
-        // In VexxHost we'll use the network called "private" for the
-        // instance NIC, attach a floating IP from the "public" network and
-        // use the v3-starter-4 instance (ram=8GiB, CPUs=4).
-        // The clouds.yaml config should be located at ${OPENSTACK_KOLA_TESTS_CONFIG}/config.
-        //
-        // Since we don't have permanent images uploaded to VexxHost we'll
-        // skip the upgrade test.
-        try {
-            fcosKola(cosaDir: env.WORKSPACE, parallel: 5,
-                     build: params.VERSION, arch: params.ARCH,
-                     extraArgs: params.KOLA_TESTS,
-                     skipBasicScenarios: true,
-                     skipUpgrade: true,
-                     platformArgs: """-p=openstack                               \
-                        --openstack-config-file=\${OPENSTACK_KOLA_TESTS_CONFIG}/config \
-                        --openstack-flavor=v3-starter-4                          \
-                        --openstack-network=private                              \
-                        --openstack-floating-ip-network=public                   \
-                        --openstack-image=${openstack_image_name}""")
-        } finally {
-            stage('Delete Image') {
-                // Delete the image in OpenStack
+            }
+
+            stage('Upload/Create Image') {
+                // Create the image in OpenStack
                 shwrap("""
-                ore openstack --config-file=\${OPENSTACK_KOLA_TESTS_CONFIG}/config \
-                     delete-image --id=${openstack_image_name}
+                # First delete it if it currently exists, then create it.
+                ore openstack --region=${region} --config-file=\${OPENSTACK_KOLA_TESTS_CONFIG}/config \
+                    delete-image --id=${openstack_image_name} || true
+                ore openstack --region=${region} --config-file=\${OPENSTACK_KOLA_TESTS_CONFIG}/config \
+                    create-image --file=${openstack_image_filepath} \
+                    --name=${openstack_image_name} --arch=${params.ARCH}
```

Maybe let's add an extra line:

```suggestion
                ore openstack --region=${region} \
                    --config-file=\${OPENSTACK_KOLA_TESTS_CONFIG}/config \
                    delete-image --id=${openstack_image_name} || true
                ore openstack --region=${region} \
                    --config-file=\${OPENSTACK_KOLA_TESTS_CONFIG}/config \
                    create-image --file=${openstack_image_filepath} \
                    --name=${openstack_image_name} --arch=${params.ARCH}
```